frontend/admin/src/js/components/dashboard/LatestRequestsTable.tsx

```diff
@@ -78,13 +58,11 @@ const LatestRequestsTable: React.FC<LatestRequestsTableProps> = ({ data }) => {
         description:
           "Title displayed on the latest requests table for the action column.",
       }),
-      width: 10,
       accessor: ({ id, fullName }) =>
-        requestActionAccessor(
-          id,
-          paths.searchRequestUpdate(id),
-          intl,
-          fullName,
+        tableViewItemButtonAccessor(
+          paths.searchRequestView(id),
+          "request",
+          fullName || "",
```

Comment:
It's actually only able to be a string or undefined. I've changed this to actually fallback to undefined though since I think it makes a bit more sense.